packages/mobile/src/web3/dataEncryptionKey.ts

```diff
@@ -170,7 +242,8 @@ export function* getAuthSignerForAccount(account: string) {
       const upToDate: boolean = yield call(
         isAccountUpToDate,
         accountsWrapper,
-        account,
+        accountAddress,
```

Comment:
I'm not totally clear when we make ODIS requests in Valora but just want to flag the additional performance cost of checking that our DEK is registered here

Reply:
Yup, I think we can remove this when we have more certainty that DEKs are being registered. I believe the reason we had this is because we placed DEK registration attemps in various spots across Valora to ensure the widest possible coverage (believe it happens on the first of invite redemption, verification, or user's first send tx) but also didn't want to charge the user for transactions they didn't need.

Good call out but don't feel super comfortable removing it at this point.